static/docs/use-cases/huge-data-on-an-external-drive.md

```diff
@@ -0,0 +1,156 @@
+# Huge Data On An External Local Drive
```

Comment:
we def should change the title, and come with better terms for "external local" thingy

Reply:
Tip: for "local remote"  in docs we explain that the term isn't contradictory because its location is external to the DVC project (thus **remote**), yet still **local** to the file system (where the workspace is located).

Reply:
Updated to https://github.com/iterative/dvc.org/pull/565/files#r315518692